include/deal.II/lac/solver_cg.h

```diff
@@ -96,6 +96,15 @@ class SolverCG : public Solver<VECTOR>
    */
   typedef types::global_dof_index size_type;
 
+  template<class T>
+  using signal=boost::signals2::signal<T>;
```

Comment:
This is a use of C++11 (template using statements didn't exist before). Please just spell out the fully qualified name wherever you use it.